src/test/java/org/jabref/logic/bibtexkeypattern/BibtexKeyGeneratorTest.java

```diff
@@ -996,105 +1042,105 @@ void testcrossrefShorttitleInitials() {
     }
 
     @Test
-    void generateKeyWithOneModifier() throws Exception {
+    void generateKeyWithOneModifier() {
         BibEntry entry = new BibEntry();
         entry.setField(StandardField.TITLE, "The Interesting Title");
-        assertEquals("theinterestingtitle", BibtexKeyGenerator.generateKey(entry, "title:lower"));
+        assertEquals("theinterestingtitle", generateKey(entry, "title:lower"));
     }
 
     @Test
-    void generateKeyWithTwoModifiers() throws Exception {
+    void generateKeyWithTwoModifiers() {
         BibEntry entry = new BibEntry();
         entry.setField(StandardField.TITLE, "The Interesting Title");
-        assertEquals("theinterestingtitle", BibtexKeyGenerator.generateKey(entry, "title:lower:(_)"));
+        assertEquals("theinterestingtitle", generateKey(entry, "title:lower:(_)"));
     }
 
     @Test
-    void generateKeyWithTitleCapitalizeModifier() throws Exception {
+    void generateKeyWithTitleCapitalizeModifier() {
         BibEntry entry = new BibEntry();
         entry.setField(StandardField.TITLE, "the InTeresting title longer than THREE words");
-        assertEquals("TheInterestingTitleLongerThanThreeWords", BibtexKeyGenerator.generateKey(entry, "title:capitalize"));
+        assertEquals("TheInterestingTitleLongerThanThreeWords", generateKey(entry, "title:capitalize"));
     }
 
     @Test
-    void generateKeyWithShortTitleCapitalizeModifier() throws Exception {
+    void generateKeyWithShortTitleCapitalizeModifier() {
         BibEntry entry = new BibEntry();
         entry.setField(StandardField.TITLE, "the InTeresting title longer than THREE words");
-        assertEquals("InterestingTitleLonger", BibtexKeyGenerator.generateKey(entry, "shorttitle:capitalize"));
+        assertEquals("InterestingTitleLonger", generateKey(entry, "shorttitle:capitalize"));
     }
 
     @Test
-    void generateKeyWithTitleTitleCaseModifier() throws Exception {
+    void generateKeyWithTitleTitleCaseModifier() {
         BibEntry entry = new BibEntry();
         entry.setField(StandardField.TITLE, "A title WITH some of The key words");
-        assertEquals("ATitlewithSomeoftheKeyWords", BibtexKeyGenerator.generateKey(entry, "title:titlecase"));
+        assertEquals("ATitlewithSomeoftheKeyWords", generateKey(entry, "title:titlecase"));
     }
 
     @Test
-    void generateKeyWithShortTitleTitleCaseModifier() throws Exception {
+    void generateKeyWithShortTitleTitleCaseModifier() {
         BibEntry entry = new BibEntry();
         entry.setField(StandardField.TITLE, "the InTeresting title longer than THREE words");
-        assertEquals("InterestingTitleLonger", BibtexKeyGenerator.generateKey(entry, "shorttitle:titlecase"));
+        assertEquals("InterestingTitleLonger", generateKey(entry, "shorttitle:titlecase"));
     }
 
     @Test
-    void generateKeyWithTitleSentenceCaseModifier() throws Exception {
+    void generateKeyWithTitleSentenceCaseModifier() {
```

Comment:
You can safely ignore the `throws Exception` for tests (although the intellj warnings are there). In fact, it's good practice to add this to test methods so that the tests don't need to change if you add an exception to the method under test.

Reply:
I don't have an opinion on this issue. I am, unfortunately, currently using Intellij recommendations instead of a brain/experience :P
Would you like me to add it to all methods in `BibtexKeyGeneratorTest` and/or should I keep it in mind for any future changes?

Reply:
It was just a minor remark to keep in mind for the next time.